packages/photoshop-cornell/config.yml

```diff
@@ -0,0 +1,41 @@
+Id:             'photoshop-cornell'
+Description:    'Adobe Photoshop CC 2018'
+Version:        '19.1.0'
+
+Install:
+    File:       '%INSTALL_DIR%\Build\setup.exe'
+    FileType:   'exe'
+    Arguments:  '--silent'
+    ExitCodes:  [0]
+
+ChocoPackages: []
+
+Applications:
+    Photoshop: 
+    DisplayName:    'Adobe Photoshop CC 2018'
+        Path:           '%WINDIR%\system32\WindowsPowerShell\v1.0\powershell.exe'
+        LaunchParams:   '-windowstyle hidden -executionpolicy bypass -file "C:\programdata\Adobe\Launcher\photoshop.ps1"'
+        WorkDir:        '%WINDIR%\system32\WindowsPowerShell\v1.0'
+
+Files:
+    '%INSTALL_DIR%\AdobeCC-2018-Lic\AdobeSerialization.exe': '%ALLUSERSPROFILE%\Adobe\Files\AdobeSrialization.exe'
+    '%INSTALL_DIR%\AdobeCC-2018-Lic\helper.bin': '%ALLUSERSPROFILE%\Adobe\Files\helper.bin'
+    '%INSTALL_DIR%\AdobeCC-2018-Lic\prov.xml': '%ALLUSERSPROFILE%\Adobe\Files\prov.xml'
+    '%INSTALL_DIR%\photoshop.ps1': '%ALLUSERSPROFILE%\Adobe\Launcher\photoshop.ps1'
+
+Environment: { }
+
+Services: { }
+
+ScheduledTasks:
+    - 'Adobe Activation'
+    - 'Update Adobe Products'
+
```

Comment:
Can you put the Scheduled task XML exports under %TOOLS_DIR% so that they can be under source control?

Also, would prefer names on files we create to have underscores or hyphens instead of spaces, and all lowercase.